app/views/layouts/_header.html.erb

```diff
@@ -1,6 +1,7 @@
+<% is_not_administrating_site = controller.class.name.to_s.split('::').first != 'Admin' %>
```

Comment:
I don't think this is necessary here. `@current_course` isn't set in the route you've added.

Reply:
Ah, yeah it isn't. I found it necessary when editing/creating courses. I'll get rid of it here.